### DISCUSSION TITLE
Q1

### DISCUSSION THREAD

**Discussion by MagDev2023** [MagDev2023/MagDev2023#3] [Q&A] [answered]
Bagaimana kita akan mengembangkan sebuah portofolio untuk komunitas agar bisa rame?

**Comment by Abito21** [comment, marked as answer]
Perlu membuat sebuah tim yang solid tanpa harus ahli bertahun", tim yang saling bekerjasama memiliki pengalaman setidaknya sedikit di industri atau dalam melakukan berbagai projek.
